src/scss/_packages.scss

```diff
@@ -147,4 +147,41 @@
     .package .content-container {
         box-shadow: 0px 4px 4px rgba(220, 220, 220, 0.25);
     }
+}
+
+.package-card {
+    box-shadow: 0px 4px 4px rgba(220, 220, 220, 0.25);
+    width: 346px;
+}
+
+.package-card-compact {
+    border-color: map-get($gray, 300);
+    box-shadow: 0px 4px 4px rgba(220, 220, 220, 0.25);
+    width: 346px;
+    @media (min-width: 768px) {
+        width: 320px;
+        top: 160px;
+        position: absolute;
+    }
+    @media (min-width: 1024px) {
+        width: 320px;
+        top: 138px;
```

Comment:
100% non-blocker, def a follow, can we add some comments here about what these specific numbers represent/what they depend on?

Reply:
thanks @kimberleyamackenzie. yeah i'll do a follow, mainly so I don't have to do the whole dance of rebuilding theme and relinking it to the other PR.